src/Skoruba.IdentityServer4.Admin.Api/Controllers/RolesController.cs

```diff
@@ -19,7 +19,7 @@ namespace Skoruba.IdentityServer4.Admin.Api.Controllers
     [Route("api/[controller]")]
     [ApiController]
     [TypeFilter(typeof(ControllerExceptionFilterAttribute))]
-    [Produces("application/json")]
+    [Produces("application/json", "application/problem+json")]
```

Comment:
This is something new to me, but apparently it's a nice way of hinting user that something is wrong and output will be in JSON. I've read about it here: https://symfonycasts.com/screencast/rest/application-problem